api/services/ocean/spark/paths/cluster.yaml

```diff
@@ -0,0 +1,35 @@
+summary: Ocean Spark
+
+get:
+  summary: "Get Cluster"
+  description: >
+    Describe an Ocean Spark cluster.
+  operationId: "OceanSparkClusterGet"
+  tags:
+    - "Ocean Spark"
+  parameters:
+    - $ref: "../../../../commons/parameters/nonRequiredAccountId.yaml"
+    - $ref: "../parameters/path/oceanSparkClusterId.yaml"
+  responses:
+    200:
+      $ref: "../responses/clusters.yaml"
+    400:
+      description: "Bad Request"
+
+delete:
+  summary: "Delete Cluster"
+  description: >
+    Delete an Ocean Spark cluster.
```

Comment:
We should add a warning that the associated ressources do not get cleaned up (as we did in the UI modal). The ocean-spark operator is uninstalled, but the cluster remains an Ocean cluster, and the EKS cluster is not deleted, right?
cc @thorsteinnth

Reply:
Done with the modal's language. thanks!